build/partials/_navbar.md

```diff
@@ -13,6 +13,8 @@
   - [Plugin reviews](guides/plugin-reviews.md)
   - [Plugin updates](guides/plugin-updates.md)
   - [Setting up a new project](guides/delivery/setting-up-a-new-project.md)
+  - [Running a discovery kick-off workshop](guides/running-a-discovery-kick-off-workshop.md)
```

Comment:
Would you mind reordering the new items so the guides are still listed alphabetically, please?

Reply:
Thanks Christina. I would argue that alphabetical is not a good order for the guides:
https://www.nngroup.com/articles/alphabetical-sorting-must-mostly-die/

Reply:
This can certainly be a topic for the playbook reorganisation project! But until we arrive at a more logical ordering, I'd argue in favour of consistency with what little order we do have.

Reply:
OK. Alphabetical order for now.